packages/passport/sdk/src/zkEvm/provider/eip6963.ts

```diff
@@ -0,0 +1,26 @@
+  name: 'Immutable Passport',
+  rdns: 'com.immutable.passport',
+  uuid: uuidv4(),
+} as EIP6963ProviderInfo;
+
+export function announceProvider(
+  detail: EIP6963ProviderDetail,
+): () => void {
+  if (!window) return () => {};
+  const event: CustomEvent<EIP6963ProviderDetail> = new CustomEvent(
+    'eip6963:announceProvider',
+    { detail: Object.freeze(detail) },
+  ) as EIP6963AnnounceProviderEvent;
+
+  window.dispatchEvent(event);
+
+  const handler = () => window.dispatchEvent(event);
+  window.addEventListener('eip6963:requestProvider', handler);
+  return () => window.removeEventListener('eip6963:requestProvider', handler);
```

Comment:
@Jon-Alonso This was the original implementation carried over from `mipd` library, it's a cleanup function to remove the listener. Within this context I don't see us needed it.. I've removed it.
But I guess we make the assumption that people aren't going to be calling connectEvm() more than once? otherwise we potentially add multiple listeners without ever cleaning it up.. and sending different instances of the provider.